### DISCUSSION TITLE
Solving the problem of sound quality via I2S output

### DISCUSSION THREAD

**Discussion by glido1** [ElectroTechnique/electrotechnique.github.io#49] [Hardware Issues]
On the USB / PC powered Tsynth there is a nightmare of interference reaching 200mV, there are even sine waveforms. It all appears on the analog output, so it is not surprising that the dynamics / sound quality is distorted. Added to this is the strange sgtl5000 analog out output impedance.

**Comment by cdw2000** [comment]
Can you elaborate with explanation of what board you are using and how it is connected to the Teensy?   Is this a board of your design?   Did you have to change the TSynth firmware to drive the new DAC, whatever it is?
